wordpress-mobile/AztecEditor-iOS
# Unknown HTML Edition

### Details:
In this PR we're adding support for Unknown HTML Edition.

cc @diegoreymendez 
Closes #411


### To test:
- Verify that the Unit Tests run correctly
- Launch the Demo and tap over the `[TABLE]` at the bottom
- Verify that the Unknown HTML Editor works smoothly

#### Known Issues:
- The new HTML should be valid. We'll address this later.
